tensorflow/tensorflow.bzl

```diff
@@ -365,6 +366,7 @@ def tf_copts(
         if_mkl(["-DINTEL_MKL=1"]) +
         if_mkldnn_openmp(["-DENABLE_ONEDNN_OPENMP"]) +
         if_enable_mkl(["-DENABLE_MKL"]) +
+        if_mkldnn_aarch64_acl(["-DENABLE_MKL", "-DENABLE_ONEDNN_OPENMP"]) +
```

Comment:
This might be redundant, if_enable_mkl and if_mkldnn_openmp should be sufficient to set up the correct defines.

Reply:
Thanks for pointing this out, I will check and correct it.

Reply:
The line
`if_mkldnn_aarch64_acl(["-DENABLE_MKL", "-DENABLE_ONEDNN_OPENMP"])`
here is actually crucial for the correct functionality.